### DISCUSSION TITLE
AVIF The difference between living and dying

### DISCUSSION THREAD

**Discussion by Stuartemk** [joomla/joomla-cms#35593] [2. Cold Features]
The AVIF format is the greatest advancement of image and video for any device on the internet, not only for web pages that includes any and all cms and it should be supported, used and implemented by default in Joomla since any cms will be at a huge disadvantage not only against big tech but also at a disadvantage against any other technology on the internet including all other cms and competitors.

**Comment by Fedik** [comment]
https://caniuse.com/?search=AVIF

**Comment by dgrammatiko** [comment]
You can use AVIF today using a 3rd PD plugin like: https://responsive-images.dgrammatiko.dev
BTW the plugin linked here was created back in 2016 as a base for the GSOC students to bring it into the core...
